### PR TITLE
add note about making bookmarks for things like monitoring tools and ssh connections

### PR DESCRIPTION
**Pull request by zadr**
Wasn't sure where to put this, and it seems to flow reasonably well in the middle of the existing on-call setup section, but the side effect that was that the following lines had their numbers changed too. Oops.

I thought about linking to steps for how to autostart ssh bookmarks on [mac](http://osxdaily.com/2012/06/03/create-ssh-bookmarks-in-terminal-for-mac-os-x/) and [ubuntu](http://superuser.com/questions/198015/open-gnome-terminal-programmatically-and-execute-commands-after-bashrc-was-execu) (well, `gnome-terminal`), but the rest of the page is platform-agnostic, so I erred on the side of sticking with the quo.

### CODE REVIEW

File: on_call_kit.md [hunk 1]
```diff
@@ -10,9 +10,10 @@ It's helpful to have a checklist that you run through prior to going on-call to
...
-7. A hands-free kit for your phone, whether this be the headphones that came with it or a bluetooth earpiece. Make sure you've tested it with your phone in advance of going on-call. It's good to keep your hands available in case you need to call up a colleague or join a conference call.
-7. A smart watch (pebble, Apple watch, etc.) can be helpful for late night alerts. The vibration of a smart watch will wake only you up where an audible alert from your phone might also wake up your partner as well. Have your charger handy just in case.
-8. A small zippable bag to keep all of your on-call kit in (other than your laptop). Your items, such as chargers, should never leave this bag unless you are using them on call. That way you never have to scramble to be ready for on-call. 
+7. Consider making bookmarks for everything. Save that group of browser tabs with the recipes for restarting things and monitoring tools ordered the way you like it. If there's a set of boxes you like to connect to, save their ssh connection info in your terminal and have everything auto-connect on launch -- 3 A.M. when everything's beeping at you isn't the time to test your memory.
```
Comment (alicegoldfuss) [inline on_call_kit.md]: Do you have a guide for auto-sshing that you can link in?

**Review by alicegoldfuss — CHANGES_REQUESTED** [review]

File: on_call_kit.md (hunk 1 above)
Comment (zadr) [reply on_call_kit.md]: sure. mentioned a few in the PR notes as possibilities (one for mac and another gnome-based env's like ubuntu), but don't know of any for KDE's `konsole`, or any of the *BSD's. can leave that blank for others to fill in, i suppose.

Comment (alicegoldfuss) [reply on_call_kit.md]: Ah, should have read the notes more thoroughly! You can just put in a parenthesis like (here's one for Mac OS and Ubuntu) and let others fill in other OSes.

**Comment by alicegoldfuss** [comment]
Thank you!

**Pull request merged**